apps/academy/src/components/mdx/LessonInformationalModal.tsx

```diff
@@ -17,7 +17,10 @@ interface Props {
 export default function LessonInformationalModal({ buttonText, title, children }: Props) {
   return (
     <Dialog>
-      <DialogTrigger className="my-3 flex rounded-2xl bg-yellow-600 p-2 font-semibold	text-white">
+      <DialogTrigger
+        className="mx-auto my-3 flex h-12	w-36 rounded-3xl bg-[#721F79] p-2 text-2xl
```

Comment:
is the space between `h-12` and `w-36` causing the text not to render properly inside the button element on the page?
![button text InformationalModal](https://github.com/Developer-DAO/academy-turbo/assets/40469149/fa3356e2-5dac-43c4-8bce-d1bc5491cdf5)